Fix Ollama not working in g4f api

hdsz25 opened the pull request:
it's not working in api, please merge only below files to fix this bug:
g4f/providers/base_provider.py
g4f/cli.py
g4f/api/__init__.py
g4f/Provider/Ollama.py

Also set default model for Ollama to use phi3:latest if available.

Thanks!

hlohaus commented:
Hey, I fixed AppConfig in the latest release.
I added args support for the model and provider in the master branch. I could add phi as the default model. Nothing else should be changed. ?

hdsz25 commented:
> Hey, I fixed AppConfig in the latest release. I added args support for the model and provider in the master branch. I could add phi as the default model. Nothing else should be changed. ?

Thanks for the quick reply. The api with parameters give below error:
![image](https://github.com/xtekky/gpt4free/assets/37258899/6e7b61c4-d8fb-4142-904e-038d39ff5dfb)

hlohaus commented:
Hey, I just updated the Bing provider in the latest version.
